index2.html

```diff
@@ -0,0 +1,1253 @@
+  </p>
+  <p>
+    The terms
+    <a href="http://dev.w3.org/html5/spec/webappapis.html#event-handlers">
+      <dfn>event handler</dfn></a> and
+    <a href="http://dev.w3.org/html5/spec/webappapis.html#event-handler-event-type">
+    <dfn>event handler event types</dfn></a> are defined in [[!HTML5]].
+  </p>
+  <p>
+    <a href='http://people.mozilla.org/~jorendorff/es6-draft.html#sec-promise-objects'>
+      <code><dfn>Promise</dfn></code></a>, and
+    <a href="https://people.mozilla.org/~jorendorff/es6-draft.html#sec-json.parse">
+      <code><dfn>JSON.parse</dfn></code></a> are defined in [[!ECMASCRIPT]].
+  </p>
+  <p>
+    The algorithms <a href="http://www.w3.org/TR/encoding/#utf-8-encode">
+    <dfn>utf-8 encode</dfn></a>, and
+    <a href="http://www.w3.org/TR/encoding/#utf-8-decode">
+    <dfn>utf-8 decode</dfn></a> are defined in [[!ENCODING]].
+  </p>
+  <p>
+    <a href="http://www.w3.org/TR/dom/#eventinit">
+      <dfn>EventInit</dfn></a>,
+    <a href="http://www.w3.org/TR/dom/#domexception">
+      <dfn>DOMException</dfn></a>,
+    <a href="http://www.w3.org/TR/dom/#aborterror">
+      <dfn>AbortError</dfn></a>,
+    <a href="http://www.w3.org/TR/dom/#syntaxerror">
+      <dfn>SyntaxError</dfn></a>,
+    <a href="http://www.w3.org/TR/dom/#notsupportederror">
+      <dfn>NotSupportedError</dfn></a>,
+    <a href="http://www.w3.org/TR/dom/#notfounderror">
+      <dfn>NotFoundError</dfn></a>, and
+    <a href="http://www.w3.org/TR/dom/#securityerror">
+      <dfn>SecurityError</dfn></a>
+    are defined in [[!DOM4]].
+  </p>
+  <p>
+    The term <dfn>web app</dfn> refers to a Web application, i.e. an application
+    implemented using Web technologies, and executing within the context of a
+    Web <a>user agent</a>, e.g. a Web browser or other Web-based runtime
+    environments.
+  </p>
+  <p>
+    The term <dfn>expressed permission</dfn> refers to an act by the user, e.g.
+    via user interface or host device platform features, via which the user
+    approves the permission of a <a>web app</a> to access the Web NFC API.
+  </p>
+  <p>
+    <code><a href="http://www.w3.org/TR/FileAPI/#blob"><dfn>Blob</dfn></a>
+    </code> is defined in [[!FILEAPI]].
+  </p>
+  <p>
+    <a href="http://heycam.github.io/webidl/#idl-DOMString">
+      <code><dfn>DOMString</dfn></code></a>,
+    <a href="http://heycam.github.io/webidl/#idl-ArrayBuffer">
+      <code><dfn>ArrayBuffer</dfn></code></a>,
+    <a href="http://heycam.github.io/webidl/#common-BufferSource">
+      <code><dfn>BufferSource</dfn></code></a> and
+    <a href="http://www.w3.org/TR/WebIDL/#idl-any">
+      <code><dfn>any</dfn></code></a>
+    are defined in [[!WEBIDL]].
+  </p>
+  <p>
+    <dfn>NFC</dfn> stands for Near Field Communications, short-range wireless
+    technology operating at 13.56 MHz which enables communication between
+    devices at a distance less than 10 cm. The NFC communications protocols and
```

Comment:
ah I guess this is what I wrote... maybe we should update the abstract to be more alike